src/utils/components/DiskModal/components/DiskTypeSelect/utils.ts

```diff
@@ -0,0 +1,12 @@
+import { V1Disk } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { diskTypes } from '@kubevirt-utils/resources/vm/utils/disk/constants';
+
+export const getDiskType = (disk: V1Disk) => {
+  if (disk.cdrom) return diskTypes.cdrom;
+
+  if (disk.lun) return diskTypes.lun;
+
+  if (disk.disk) return diskTypes.disk;
+
+  return diskTypes.disk;
```

Comment:
u can make an obj or a proxy with default value instead of those if's, there are example in our code base